airbyte-integrations/connectors/source-s3/setup.py

```diff
@@ -12,6 +12,9 @@
     "wcmatch==8.2",
     "dill==0.3.4",
     "pytz",
+    "fastavro==1.4.11",
+    "anyio==3.5.0" # needed?
```

Comment:
can we:

- remove this if not needed
- if it is, it's missing a comma

Reply:
Deleted anyio as it's not needed